### ISSUE TITLE
primefaces-spring-boot-starter 5.2.3 missing primefaces version in pom.xml

### ISSUE THREAD

**Issue by dengbingaliyun** [joinfaces/joinfaces#2197]
I have updated to version 5.2.3 today . 
But the primefaces relevant dependency could not be found in my project. So I've located the pom.xml of primefaces-spring-boot-starter 5.2.3 in my maven repository , and saw the primefaces version was missing list below:

```
<dependency>
      <groupId>org.joinfaces</groupId>
      <artifactId>cdi-spring-boot-starter</artifactId>
      <version>5.2.3</version>
      <scope>compile</scope>
      <exclusions>
        <exclusion>
          <groupId>jakarta.annotation</groupId>
          <artifactId>jakarta.annotation-api</artifactId>
        </exclusion>
      </exclusions>
    </dependency>
    <dependency>
      <groupId>jakarta.validation</groupId>
      <artifactId>jakarta.validation-api</artifactId>
      <version>3.0.2</version>
      <scope>runtime</scope>
    </dependency>
```

**Comment by persapiens** [comment]
hi @dengbingaliyun ,

thanks for informing us.

I confirm that primefaces-spring-boot-starter 5.2.3 pom artifact is broken. I receive the message from maven

```
The POM or org.joinfaces:primefaces-spring-boot-starter:5.2.3 is invalid, transitive dependencies (if any) will not be available, enable debug logging for more details
```

I think it happens because primefaces-spring-boot-starter pom version 5.2.3 does **not** contain primefaces version.

```xml
    <dependency>
      <groupId>org.primefaces</groupId>
      <artifactId>primefaces</artifactId>
      <classifier>jakarta</classifier>
      <scope>compile</scope>
    </dependency>
    <dependency>
      <groupId>org.primefaces.extensions</groupId>
      <artifactId>primefaces-extensions</artifactId>
      <classifier>jakarta</classifier>
      <scope>compile</scope>
    </dependency>
```

primefaces-spring-boot-starter pom version 5.2.2 **contains** primefaces version.

```xml
    <dependency>
      <groupId>org.primefaces</groupId>
      <artifactId>primefaces</artifactId>
      <version>13.0.4</version>
      <classifier>jakarta</classifier>
      <scope>compile</scope>
    </dependency>
    <dependency>
      <groupId>org.primefaces.extensions</groupId>
      <artifactId>primefaces-extensions</artifactId>
      <version>13.0.4</version>
      <classifier>jakarta</classifier>
      <scope>compile</scope>
    </dependency>
```

@larsgrefer , could you help us with this issue?

I look forward to your answer.

**Comment by persapiens** [comment]
@larsgrefer , I built joinfaces branch 5.2.x locally and primefaces-spring-boot-starter pom version 5.2.x-SNAPSHOT does not contain primefaces version.

**Comment by larsgrefer** [comment]
The change seems to be caused by a difference between Gradle 8.4 and 8.5/8.6

With Gradle 8.4 the pom is generated with versions. I've downgraded to Gradle 8.4 as a first fix.

**Comment by persapiens** [comment]
Great news.

Should we release joinfaces 5.2.3.1 and 5.1.9.1?

Also, should we release joinfaces 5.3.x milestones (5.3.0-m2)? If positive, I imagine the 5.3.x branch should be the default branch and tracked by dependabot.

**Comment by persapiens** [comment]
The problem happened with PrimeFaces starter, but it didn't occur with others like Togabo or Rewrite.

**Comment by larsgrefer** [comment]
> The problem happened with PrimeFaces starter, but it didn't occur with others like Togabo or Rewrite.

primefaces is a bit special, because it uses the `jakarta` classifier

**Comment by larsgrefer** [comment]
I've opened https://github.com/gradle/gradle/issues/28225

**Comment by dengbingaliyun** [comment]
@persapiens  Sorry, I posted the pom code is wrong.It shoud be like yours.

**Comment by larsgrefer** [comment]
This should be fixed with Gradle 8.7.
So we can stay on the downgraded Gtadle 8.4 until 8.7 is available.
